src/types/index.ts

```diff
@@ -201,8 +221,8 @@ export type MongoBookmark = {
   _id: ObjectId
   url: string
   label?: string
-  cmsId?: string
-  isRemoved?: boolean
+  cmsId?: string | null
+  isRemoved?: boolean | null
```

Comment:
Exactly -- I thought the same thing. By adding `| null` we can properly type our test data now, which matches what is created in MongoDB.  If there is no value passed in (since it's optional), MongoDB auto-fills that value to be null. So this gets our code to match what's actually happening in the db.